_posts/2017-11-20-owasp-ctf.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Mini-CTF OWASP
+tags: ctf
+---
+
+## Mini-CTF OWASP
+
+Présentation de 3 défis du Mini-CTF OWASP. Au menu, des techniques
+d’injection SQL, du contournement de protection de téléversement de fichier et une
+introduction à l’exploitation de unserialize PHP.
+
+Nous avons un nombre limité de places disponibles, [**l'inscription est
```

Comment:
We tend to strip all the meta stuff for the archives (that's why rename detection is not working). Eventbrite links, where, when, how.